packages/core/contracts/insured-bridge/BridgeDepositBox.sol

```diff
@@ -31,7 +31,7 @@ interface TokenLike {
  * @notice Accepts deposits on Optimism L2 to relay to Ethereum L1 as part of the UMA insured bridge system.
  */
 
-abstract contract BridgeDepositBox is Legacy_Testable, Legacy_Lockable {
+abstract contract BridgeDepositBox is Testable, Lockable {
```

Comment:
One problem with this change is it blocks us from testing on the current optimism test net. This should be fine for now(we can just use the code before this pr). 

I would ask, though, that we get all remaining across PRs merged in before merging this PR. In particular, @mrice32 has two important pending prs that we should merge first. This way we can check out the code before this pr and continue to test while we wait for the next version of optimism to be launched